playlist.js

```diff
@@ -53,6 +58,7 @@ function playGame(metadata, difficulty) {
         } else {
           snackNotif('Challenge Complete', 5000);
           document.getElementById('playlist-ready').innerHTML = 'Challenge Complete!';
+          document.getElementById('start-challenge').disabled = false;
```

Comment:
see if this makes sense. disabling button until current game is over

Reply:
it looks good to me